int8 steering
---
bool was_set
